molecule_vagrant/modules/vagrant.py

```diff
@@ -345,6 +348,11 @@ def __init__(self, module):
         self._module = module
         self.provision = self._module.params["provision"]
         self.cachier = self._module.params["cachier"]
+        self.provider_name = self._module.params["provider_name"]
+        # Keep historic behaviour and use virtualbox as "default".
+        # It's important in order to be able to set vm memory/cpus.
+        if self.provider_name is None:
+            self.provider_name = "virtualbox"
```

Comment:
But this assumption is not too ideal? Because if we really pass in with`self.provider_name = None`, it will never passing as undefined to Vagrant?